core-web/libs/block-editor/src/lib/extensions/ai-content-actions/ai-content-actions.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Observable, of } from 'rxjs';
+
+import { Component, EventEmitter, Output, OnInit } from '@angular/core';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+interface ActionOption {
+    label: string;
+    icon: string;
+    selectedOption: boolean;
+    callback: () => void;
+}
+
+export enum ACTIONS {
+    ACCEPT = 'ACCEPT',
+    DELETE = 'DELETE',
+    REGENERATE = 'REGENERATE'
+}
+
+@Component({
+    selector: 'dot-ai-content-actions',
+    templateUrl: './ai-content-actions.component.html',
+    styleUrls: ['./ai-content-actions.component.scss']
+})
+export class AIContentActionsComponent implements OnInit {
+    @Output() actionEmitter = new EventEmitter<ACTIONS>();
+
+    actionOptions!: ActionOption[];
+
+    constructor(private aiContentService: AiContentService) {}
+
+    ngOnInit() {
+        this.actionOptions = [
+            {
+                label: 'Accept',
```

Comment:
What are we with the translation?

Reply:
There is no support for translation implemented yet